package.json

```diff
@@ -28,13 +28,16 @@
   "dependencies": {
     "aws-auto-auth": "^0.1.1",
     "aws-sdk": "^2.1.31",
+    "commander": "^2.8.1",
     "csv-to-vrt": "0.0.2",
+    "end-of-stream": "^1.1.0",
     "fs-extra": "^0.18.4",
-    "hyperquest": "^1.2.0",
-    "s3-upload-stream": "^1.0.7",
-    "unzip": "^0.1.11"
-  },
-  "devDependencies": {
-    "tape": "^4.0.0"
+    "ftp": "^0.3.10",
```

Comment:
Why the move to `devDependencies`?  Other than `tape`, most of these seem like they should go under `dependencies`.

Also, where is `end-of-stream` used?  I see its a dependency of `pump`.  Is this just to get a newer or specific version compared to comes with `pump`?  Or is this somehow necessary for the shrinkwrappings?